scripts/masternode.js

```diff
@@ -271,11 +271,20 @@ export default class Masternode {
     }
 
     /**
-     * @return {Promise<Array>} A list of currently active proposal
+     * 
+     * @param {object} options
+     * @param {bool} options.fAllowFinished - Pass `true` to stop filtering proposals if finished
+     * @return {Promise<Array<object>} A list of currently active proposal
      */
-    static async getProposals() {
+    static async getProposals({ fAllowFinished = false }) {
```

Comment:
```suggestion
    static async getProposals({ fAllowFinished = false } = {}) {
```
Add default parameter value